ruby1.8.7.gemfile

```diff
@@ -0,0 +1,22 @@
+source 'http://rubygems.org/'
+
+gemspec
+
+gem 'addressable', '< 2.3.6'
```

Comment:
Ah thanks, that was a mistake! coding too late at night. I'm pushing up the fix now